notebooks/tutorials/hello-syft/01-hello-syft.ipynb

```diff
@@ -11,6 +11,45 @@
     "# Hello Syft"
```

Comment:
<p>use node.login. I think the login bug was fixed so we shouldnt have to login again? Could you verify that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Verified that using both clients without re-login now works.</p>